### DISCUSSION TITLE
How to set height of map (ipyleafmap) in solara?

### DISCUSSION THREAD

**Discussion by rururu** [widgetti/solara#289] [Q&A]
Dear Solara users and team!

How to set height of map (ipyleafmap) in solara?
Any help would be greatly appreciated.

Sincerely,
  Ru

**Comment by maartenbreddels** [comment]
Hi Ruslan,

Good question,

I was looking at:
 * https://github.com/widgetti/solara/blob/a3f9051aee40ba91c5026b327010acbb3f6d3dcb/solara/website/pages/examples/libraries/ipyleaflet_advanced.py#L34

And increased the height, but that did not work. But combined with the following, the ipyleaflet map grows to the parent height.
```python
        ipyleaflet.Map.element(
            layout={"height": "100%"},
            zoom=zoom.value,

```

Does that help you?

Regards,

Maarten

**Reply by rururu** [reply]
Works perfectly! Thank you very much, Maarten.
